ui/app/styles/components/tabs.scss

```diff
@@ -69,3 +71,7 @@
     outline: none;
   }
 }
+
+.tabs:not(:last-child) {
+  margin-bottom: $size-4;
```

Comment:
Hm, not sure we want a margin-bottom on the tabs. For example, here's what I'm seeing in transit: 
<img width="856" alt="Screenshot 2023-04-11 at 4 22 23 PM" src="https://user-images.githubusercontent.com/82459713/231291349-1fbf1365-315b-45eb-93a4-7f014c3d480e.png">

Reply:
Ah, good catch. This was for a specific use case. I'll track it down and amend.

Reply:
Is it the space between the bottom line and the "Type" label that needs to be ammended? I would add that to the form underneath the tabs, not the tabs themselves.